src/js/discharge-wizard/reducers/discharge-wizard.js

```diff
@@ -2,51 +2,49 @@ import { DW_UPDATE_FIELD } from '../actions';
 import _ from 'lodash';
 
 const initialState = {
-  '1_reason': null, // 1
-  '2_dischargeType': null, // 1a
-  '3_intention': null, // 1b
-  '4_dischargeYear': undefined, // 2
-  '5_dischargeMonth': undefined, // 2a
-  '6_courtMartial': null, // 3
-  '7_branchOfService': null, // 4
+  '1_branchOfService': null, // 4
```

Comment:
Not sure what the comments mean. They don't seem to correlate to the question ordering in the previous version either.